lib/http.js

```diff
@@ -40,10 +39,7 @@ exports.get = function(options, cb) {
 exports._connectionListener = server._connectionListener;
 const Server = exports.Server = server.Server;
 
-exports.createServer = function(requestListener) {
-  return new Server(requestListener);
-};
-
+exports.createServer = requestListener => new Server(requestListener);
```

Comment:
I don't think we have one. The eslint rule would be http://eslint.org/docs/rules/arrow-parens.html